anymal_rbprm/__init__.py

```diff
@@ -0,0 +1,5 @@
+from pathlib import Path
+
+def prefix():
```

Comment:
Could you add a second blank line here ?

Reply:
Done